kickstarts/base.ks.erb

```diff
@@ -1,5 +1,8 @@
 # Kickstart file automatically generated by anaconda.
+
 # Run the text based installer to see dependency info in screenshots
+text
```

Comment:
@jrafanie you can certainly use VNC to connect to a VM that is in text mode. The VNC server is provided by the hypervisor not the guest.